time_manager/time_manager.F90

```diff
@@ -3247,115 +2645,4 @@ subroutine time_list_error (T,Terr)
-!        next_rad_time = time + set_time(7200, 0)
-!
-!   ! Test repeat alarm
-!        repeat_alarm_freq = set_time(0, 1)
-!        repeat_alarm_length = set_time(7200, 0)
-!
-!   ! Loop through a year
-!        do i = 1, num_steps
-!
-!   ! Increment time
-!        time = time + dt
-!
-!   ! Test repeat alarm
-!        if(repeat_alarm(time, repeat_alarm_freq, repeat_alarm_length)) &
-!        write(*, *) 'REPEAT ALARM IS TRUE'
-!
-!   ! Should radiation be computed? Three possible tests.
-!   ! First test assumes exact interval; just ask if times are equal
-!   !     if(time == next_rad_time) then
-!   ! Second test computes rad on last time step that is <= radiation time
-!   !     if((next_rad_time - time) < dt .and. time < next_rad) then
-!   ! Third test computes rad on time step closest to radiation time
-!         if(interval_alarm(time, dt, next_rad_time, set_time(7200, 0))) then
-!           call get_date(time, years, months, days, hours, minutes, seconds)
-!           write(*, *) days, month_name(months), years, hours, minutes, seconds
-!
-!   ! Need to compute real number of days between current time and astro_base
-!           call get_time(time - astro_base_date, seconds, days)
-!           astro_days = days + seconds / 86400.
-!   !       write(*, *) 'astro offset ', astro_days
-!        end if
-!
-!   ! Can compute daily, monthly, yearly, hourly, etc. diagnostics as for rad
-!
-!   ! Example: do diagnostics on last time step of this month
-!        call get_date(time + dt, years, months2, days, hours, minutes, seconds)
-!        call get_date(time, years, months, days, hours, minutes, seconds)
-!        if(months /= months2) then
-!           write(*, *) 'last timestep of month'
-!           write(*, *) days, months, years, hours, minutes, seconds
-!        endif
-!
-!   ! Example: mid-month diagnostics; inefficient to make things clear
-!        length = days_in_month(time)
-!        call get_date(time, years, months, days, hours, minutes, seconds)
-!        mid_date = set_date(years, months, 1) + set_time(0, length) / 2
-!
-!        if(time < mid_date .and. (mid_date - time) < dt) then
-!           write(*, *) 'mid-month time'
-!           write(*, *) days, months, years, hours, minutes, seconds
-!        endif
-!
-!        end do
-!
-!    </PRE>
-!   end program time_main2
-
-!   </TESTPROGRAM>
-!   <NOTE>
-!     The <a name="base date">base date</a> is implicitly defined so users don't
-!     need to be concerned with it. For the curious, the base date is defined as
-!     0 seconds, 0 minutes, 0 hours, day 1, month 1, year 1
-!   </NOTE>
-!   <NOTE>
-!     Please note that a time is a positive definite quantity.
-!   </NOTE>
-!   <NOTE>
-!     See the <LINK SRC="TEST PROGRAM">Test Program </LINK> for a simple program
-!     that shows some of the capabilities of the time manager.
-!   </NOTE>
-! </INFO>
-!> @}
-! close documentation grouping
+end module time_manager_mod
```

Comment:
Is there an unprintable character at the end of this file?  I see a red circle with a line through it in the GitHub PR